circus-train-core/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
This is the magic to allow Mockito to mock final classes (needed in `ManifestAttributesTest`) so we don't need PowerMock any more, for the curious - https://www.baeldung.com/mockito-final